fs-sim/src/System/FS/Sim/MockFS.hs

```diff
@@ -46,6 +46,7 @@ module System.FS.Sim.MockFS (
   , renameFile
     -- * Exported for the benefit of tests only
   , Files
+  , openHandles
```

Comment:
I wonder if this is the right place to put this export. I interpret this section as exported only for the benefit of `fs-sim:test`, not tests in general

Reply:
Right, fs-sim itself is all for testing, so probably it can go anywhere else.

Reply:
Let's put it next to `numOpenHandles`